spring-cloud-alibaba-starters/spring-cloud-alibaba-governance/resource-transform/opensergo-resource-transform/src/main/java/com/alibaba/cloud/governance/opensergo/OpenSergoConfigProperties.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2013-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.cloud.governance.opensergo;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(OpenSergoConfigProperties.PREFIX)
+public class OpenSergoConfigProperties {
+
+	/**
+	 * Prefix in yaml.
+	 */
+	public static final String PREFIX = "spring.cloud.opensergo";
+
+	/**
+	 * Configurations about OpenSergo Server Endpoint.
+	 */
+	private String endpoint;
+
+	/**
+	 * Namespace Configuration about OpenSergo Config.
+	 */
+	private String namespace = "default";
```

Comment:
Should we need to  support several endpoints or several namespaces here. If need, we can use Collections, like following:
``` java
private List<OpenSergo> opensergoConfigList
```
   
OpenSergo.class

``` java 
// name for endpoint
private String name;

// endpoint
private String endpoint;

// muti-namespace in endpoint
// namespaceMap[name, namespace]
private Map<String, String> namespaceMap;
```

user can distinguish them by `name for endpoint` and `name in namespaceMap`, and use them more flexible